src/makim/makim_cli.py

```diff
@@ -0,0 +1,203 @@
+"""
```

Comment:
@abhijeetSaroha 

Why you are creating a new module, instead of using cli.py?

Reply:
For now, I have created a new module. When it will complete, I will remove the previous `cli.py` and change this one to `cli.py` . 
I do this so that functioning of makim continue until new cli with typer fully created.

But if you want I will remove the previous one.

Reply:
yep .. feel free to remove it ... you always can check the current version on github .. so no worries about that file :)